proposed/tracing-meta.md

```diff
@@ -0,0 +1,49 @@
+# Application Tracing Meta Document
+
+## 1. Summary
+
+Application performance monitoring (APM) tools are becoming increasingly popular, however, the lack of formal interfaces is holding back the ecosystem.
+Whilst developers are inherently free to use whichever tools suit their use-cases, this is not the case for framework and library creators/maintainers.
+
+This standard aims to create an intentionally minimalist set of interfaces that can be used to provide tracing signals to 3rd party libraries in a unified manner.
+
+## 3. Scope
+
+### 3.1 Goals
+
+* To provide a set of interfaces for library and framework developers to add tracing signals to their codebase.
+  This would in turn allow other APM providers to receive the same signals for further processing or analysis.
+* To allow traces collected by various APM libraries to be reused by other libraries/tools.
+* This PSR may provide a minimal `TraceProvider`, etc. for other APM tools to extend, should they choose.
+
```

Comment:
```suggestion
* To provide a set of interfaces for HTTP client implementations to enable sending outgoing distributed trace headers.
```

I think that an important goal of the spec should also be to provide interfaces to support distributed tracing. I see it as separate to providing tracing signals, and I think it's more widely useful. I believe most/all APM implementations support the [W3C traceparent+tracestate headers](https://www.w3.org/TR/trace-context/) (I checked elastic, datadog, and new relic), but they sometimes leave the injection of outgoing trace headers as an exercise for the reader. Having a PSR interface that was implemented by popular http client implementations would make tracing easier and more useful, IMO.

Reply:
I'm not sure it fits this proposal: distributed tracing is surely useful and we need to provide interfaces to support it, but the choice of decorating an http request with tracing header should be an implementation detail. 

Additionally this proposal is not an extension of PSR-18 and it shouldn't be from my point of view.

Surely http clients should be aware of the problem, but whether to decorate a request and how to do it is little bit out of scope at the moment.

Reply:
That makes sense - perhaps then something lighter like `To expose distributed tracing information to other libraries/tools`? I guess the important part is a consistent way to get those values from a tracing implementation. If we think that the existing wording sufficiently covers this, then I'm happy with the goals as they are.

Reply:
Yes, much better to me.
Maybe is already covered by `To allow traces collected by various APM libraries to be reused by other libraries/tools.`, but we can explicit it to make it crystal clear.

Reply:
I believe I have added clarification by way of the definitions in 916f0c2
Please do let me know if you disagree, etc